ext/hal/atmel/asf/sam/include/same70/component/gmac.h

```diff
@@ -4852,14 +4852,14 @@ typedef struct {
   __I  uint32_t GMAC_PEFRSL;    /**< (GMAC Offset: 0x1F8) PTP Peer Event Frame Received Seconds Low Register */
   __I  uint32_t GMAC_PEFRN;     /**< (GMAC Offset: 0x1FC) PTP Peer Event Frame Received Nanoseconds Register */
   __I  uint32_t Reserved6[128];
-  __I  uint32_t GMAC_ISRPQ[2];  /**< (GMAC Offset: 0x400) Interrupt Status Register Priority Queue  (index = 1) 0 */
```

Comment:
Microchip provides separate headers for revision A and B. And revision B has some more registers related low power idle. How shall I add the upstream version?

Reply:
@dgloeck: I'm not sure. I'd suggest to skimming thru https://docs.zephyrproject.org/latest/contribute/#identifying-contribution-origin to see what formal requirements the Zephyr project states, and if that woldn't answer your questions, ping maintainers like @nashif or @galak .

Note that your contribution is much appreciated, and it's not a sinecure to make it harder, I hope you agree that without keeping it organized, with the amount of boards supported by Zephyr, ext/ will turn into unmaintainable mess. The baseline requirements is as I wrote: a) separate commit; b) info in the commit message where it comes from and assessing for how long it should stay as a patch in Zephyr repo.

Reply:
It's more of a technical question, so the contribute documentation does not help. I'm fine with replacing the gmac headers with the rev B headers, but it might cause people to use the LPI registers without checking which revision they are running on. And it's a bit unclean to replace only those headers. There are more changes in other components and to me it looks like the CAN controller now uses an incompatible register layout.

Reply:
Well, I can't help much with this atmel-specific topic. @mnkp is the author/maintainer of eth_sam_gmac.c, you see that @aurel32 is active contributor, you can `git log` other files/dirs related to atmel chips (first of all, folks who imported  ext/hal/atmel/ in the first place) and call to all those people to discuss it together.

Reply:
@dgloeck I have just posted the update to the latest version for rev A in PR #12750.